examples/client/Locomotion/src/pages/Profile/Phone.js

```diff
@@ -80,6 +77,7 @@ const Phone = ({ navigation }) => {
       if (!isDevSettingOn()) {
         await AppSettings.destroy();
       }
+      await Auth.updateCaptchaToken(captchaToken);
```

Comment:
https://github.com/Autofleet/locomotion/pull/833/files#diff-c05e09a619ce22ac8b2863115ae42f257ce102e5c7aa326a1f36740dd7f1b95fR78 is deleting the token from the storage. moved it to be after the destroy